.github/workflows/e2e-tests.yaml

```diff
@@ -2,7 +2,7 @@ name: E2E tests
 
 on:
   schedule:
-    # every work day at 5:00 AM
+    # every work day at 00:00 AM
```

Comment:
should we just say midnight? what time zone?

Reply:
00:00 is more clear to me than midnight, just IMO
I will add the timezone 👍

Reply:
Updated